README.rst

```diff
@@ -320,6 +331,45 @@ current user; and ``"OS_ROOT"`` is used to open the journal from directories rel
 directory path or file descriptor. Multiple flags can be OR'ed together using a list:
 ``["LOCAL_ONLY", "CURRENT_USER"]``.
 
+``secret_filters`` (default ``[]``)
+
+Secret filters can be used to redact sensitive data which matches known patterns in logs before forwarding the message along
+to it's final destination. To use: add a number of filters following the pattern below to the reader config. The ``pattern`` is a standard
+python regex, and the matching substring will be subbed with ``redaction_message``. Patterns are compiled at runtime.
+
+Simple pattern example:
+
+This simple pattern should be used for most cases.
+
+"secret_filters": [
+  {
+    "pattern": "SECRET",
+    "redaction_message": "[REDACTED]"
+  }
+]
+
+Complex pattern example:
+
+For more complex requirements, a python regex with capture groups can be provided, and the contents of the message restructured using backrefs.
+
+"secret_filters": [
+  {
+    "pattern": "(.*?\\s?)(SECRET)(\\s?.*)",
+    "redaction_message": "\\1[REDACTED]\\3"
+  }
+]
+
+These examples will both match a secret pattern of ``SECRET``, whether it is at the start, middle or end of a log entry.
+It will also leave whitespace surrounding the matched group intact. The substring will be replaced inline with the configured
+redaction message. In this Example, a log entry of "This line contains a SECRET" will be translated to "This line contains a [REDACTED]"
+before being forwarding to any of the readers configured senders.
+
+
+``secret_filter_metrics`` ( default: ``false``)
```

Comment:
This is fine if as such but just wondering if being able to disable metrics output is especially useful. This should result in very low volume metric output so hard to see that being especially problematic for any use case.

Reply:
@mble requested this be behind a config flag, so i put i behind a config flag, But I'm open to either option here.